app/src/main/kotlin/com/simplemobiletools/calculator/helpers/MoneyCalculatorImpl.kt

```diff
@@ -86,14 +86,10 @@ class MoneyCalculatorImpl(calculator: Calculator,taxCalculator: TaxCalculator, v
         mCallback!!.setValue(value, context)
     }
 
-    fun calculateTax(){
-        //Code to ping location here
-        if(false){
-
-        }else{
-            tCallback!!.spawnTaxModal()
-        }
-    }
+    //TODO: Refactor this
+//    fun calculateTax(){
+//        tCallback!!.spawnTaxModal()
+//    }
```

Comment:
Uncomment this line

Reply:
This method as well as the TaxCalculator should be removed completely. I will refactor this in my next commit.